scss/comments.scss

```diff
@@ -41,130 +20,10 @@ $purple:                #7a43b6 !default;
         margin-bottom: 0;
     }
 
-    .comments-list {
+    ol.comments-list {

```

Comment:
This is something I missed, but basically we should never be declaring selectors via their elements (only their classes) to ensure the HTML is able to be modified if it is someway incorrect. I'll remove it now.